### PR TITLE
add rules for oc 4.3

### PR DESCRIPTION
**Pull request by xiaojiey**
@xingxingxia could you pls help to review? Thanks.

### CODE REVIEW

File: lib/rules/cli/4.3.yaml [hunk 1]
```diff
@@ -0,0 +1,1353 @@
+---
```
Comment (liangxia) [inline lib/rules/cli/4.3.yaml]: @xiaojiey  We already have 4.x rules.
```
[lxia@ocp verification-tests]$ ls lib/rules/cli/4.*
lib/rules/cli/4.1.yaml  lib/rules/cli/4.2.yaml  lib/rules/cli/4.3.yaml  lib/rules/cli/4.4.yaml
```

Any commands/options that you need does exist ? Add it into 4.1.yaml

Comment (xingxingxia) [reply lib/rules/cli/4.3.yaml]: @xiaojiey yes, agree with @liangxia

Comment (xingxingxia) [reply lib/rules/cli/4.3.yaml]: ls -l ls lib/rules/cli/4.*, you will see symlinks

**Comment by xiaojiey** [comment]
Not needed. Replaced by another PR https://github.com/openshift/verification-tests/pull/651

**Pull request closed without merging**